java/sage/media/sub/SubtitleHandler.java

```diff
@@ -121,7 +134,11 @@ else if (rv.length() == 0 && willDisplay)
         }
       }
     }
-    //System.out.println("Got Subtitle text=" + rv + " subEntryPos=" + subEntryPos);
+    finally
+    {
+      subtitleLock.readLock().unlock();
+    }
+    System.out.println("Got Subtitle text=" + rv + " subEntryPos=" + subEntryPos);
```

Comment:
Good idea. I actually forgot that debug line was originally commented out. It was very helpful during debugging.